src/lib/cellEventFactory.js

```diff
@@ -340,6 +340,12 @@ var cellEventProperties = Object.defineProperties({}, { // all props non-enumera
         }
     },
 
+    mousePointInButton: {
```

Comment:
do we have information on the button template as well?
how about "mousePointInCenterClickRect"?

Reply:
I never figured out what mousePointInCenterClickRect actually does. But this covers the case of both ButtonTemplate (mousePointInLeftClickRect) and the Increment buttons (both mousePointInLeftClickRect and RightClickRect)

Reply:
I think we should add that into the check.

imagine we are going to add one more template, where is takes the center location, then whoever created the template will should set "mousePointInCenterClickRect" to true, which should also fall into the case of "clicking on the button"

otherwise he might need to go back here and make another hypergrid change or use left/right instead.